### PR TITLE
Example refactor - remove support for OCP 4.10

### PR DESCRIPTION
**Pull request by ocofaigh**
### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,48 @@
...
   cluster_name         = "example-cluster-name"
-  # modify the value for resource_group_id with and id of a group you own
-  resource_group_id    = "id of existing resource group"
+  resource_group_id    = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
   region               = "us-south"
   force_delete_storage = true
-  vpc_id               = "id of existing VPC"
-  ## obtain the below values from the targeted VPC and adjust to the number of zones,
-  ## subnets, subnet name, cidr_block, id, zone
+  vpc_id               = "79cxxxx-xxxx-xxxx-xxxx-xxxxxXX8667"
```
Comment (rajatagarwal-ibm) [inline README.md:44]: can we remove digits?

File: examples/basic/main.tf [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 }
 
-##############################################################################
-# Create a VPC with single subnet and zone, and public gateway
-# NOTE: this is a very simple VPC/Subnet configuration for example purposes only,
-# that will allow all traffic ingress/egress by default.
-# For production use cases this would need to be enhanced by adding more subnets
-# and zones for resiliency, and ACLs/Security Groups for network security.
-##############################################################################
+########################################################################################################################
```
Comment (rajatagarwal-ibm) [inline examples/basic/main.tf:13]: We can remove this comment as all the information is provided in the example's readme

**Review by rajatagarwal-ibm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ocofaigh) [reply README.md:44]: why? I did this to show its not a secret value.

File: examples/basic/main.tf (hunk 2 above)
Comment (ocofaigh) [reply examples/basic/main.tf:13]: Well this is more info than the readme. I purposely have it in here incase anyone goes to copy and paste the VPC part, the code comment will warn them its not for production. We need to add more code comments to our examples

**Pull request merged**